playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -66,6 +69,9 @@
       {{ oc_cmd }} patch dc/docker-registry -n default -p
       '{"spec":{"template":{"spec":{"containers":[{"name":"registry","image":"{{ registry_image }}"}]}}}}'
       --api-version=v1
+    # AUDIT:changed_when_note: `false` not being set here. What we
```

Comment:
same question here regarding `changed_when` note. since this is a `command` task, won't this always return `changed_when: true` ?